app/controllers/concerns/set_for_end_of_month.rb

```diff
@@ -3,7 +3,17 @@ def set_for_end_of_month
     # Defaults to current month if it's the last week of the month.
     @show_current_month = ((Date.current.end_of_month - Date.current).to_i <= 7)
 
-    @for_end_of_month ||= if params[:for_end_of_month]
+    Sentry.configure_scope do |scope|
+      user = @current_admin || @current_user
+      scope.set_user(id: user.id)
```

Comment:
It seems like we don't have the user id for any dashboard controllers. Shall we just add it at `AdminController`?